test/jdk/java/awt/TrayIcon/TrayIconScalingTest.java

```diff
@@ -84,15 +82,13 @@ public static void main(String[] args) throws InterruptedException,
             tray.remove(icon);
         }
     }
-
```

Comment:
There's usually a blank line between methods to visually separate the end of the previous method from the start of the following one.